src/resources/audit/config.tsx

```diff
@@ -0,0 +1,24 @@
+import * as constants from '../../constants'
+
+export const availableResources = [
+  constants.R_USERS,
+  constants.R_ITEMS,
+  constants.R_BATCHES,
+  constants.R_DESTRUCTION,
+  constants.R_VAULT_LOCATION,
+  constants.R_DISPATCH,
+  constants.R_PROJECTS,
+  constants.R_MEDIA_TYPE,
+  constants.R_PLATFORMS
+]
+export const cosmeticLabels = {
+  [constants.R_USERS]: 'User',
+  [constants.R_ITEMS]: 'Item',
+  [constants.R_BATCHES]: 'Batch',
+  [constants.R_DESTRUCTION]: 'Destruction',
+  [constants.R_VAULT_LOCATION]: 'Vault Location',
+  [constants.R_DISPATCH]: 'Dispatch',
+  [constants.R_PROJECTS]: 'Project',
```

Comment:
the value of Project needs to come from the config-file, please @TahaKhanAbdalli

Reply:
got it.
I was thinking to add config based on audit and other files but never-mind.  let me add them to the constants file.

Reply:
But note: the resource used in the filter needs to be `project` (since that's the resource/table name), but the displayed label will come from the `ConfigData` (it's `Projekt` in our mock dataset)

Reply:
Should I turn `project` to `Projekt` into the resource filter?